components/loki.libsonnet

```diff
@@ -208,6 +208,13 @@ local k = (import 'ksonnet/ksonnet.beta.4/k.libsonnet');
     service.mixin.metadata.withLabels(commonLabels),
 
   components:: {
+    compactor: {
+      withReadinessProbe: true,
+      resources: {
+        requests: { cpu: '100m', memory: '100Mi' },
+        limits: { cpu: '200m', memory: '200Mi' },
```

Comment:
Perhaps this is my Thanos experience coming out, but don't you think that 200Mi memory limit for the compactor is way too small?

Reply:
I'm not familiar with how the Loki compactor works, but if it's _anything_ like the Thanos compactor I'm used to it's going to need a lot more memory.

Reply:
In general the requests/limits here are **not** sane defaults, because we are using only a free-tier CircleCI instance. They are set to such low values for E2E testing only.